src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -99,4 +100,59 @@ private class UTIL_RecordTypes_TEST {
         }
     }
 
+    //Helper Methods
+        /**
```

Comment:
Indentation is off.
Let's have a line separation from the helper method comment.